sklearn/inspection/_partial_dependence.py

```diff
@@ -908,6 +915,7 @@ def plot(self, ax=None, n_cols=3, line_kw=None, contour_kw=None):
         lines_ravel = self.lines_.ravel(order='C')
         contours_ravel = self.contours_.ravel(order='C')
 
+        vlines = np.empty_like(self.axes_).ravel()
```

Comment:
To keep with the style of the surrounding code:

```py
self.vlines_ = np.empty_like(self.axes_)
vlines_ravel = self.vlines_.ravel(order='C')
```

(Yes the order is C by default, but I recall a comment wanting this to be explicit)

Reply:
Given https://github.com/scikit-learn/scikit-learn/issues/15788, we should specific the dtype as object as well.

Reply:
OK I'll make it explicit, though note that`empty_like` will explicitly set the dtype  to that of `axes_`